deploy/helm/microfunctions/templates/_helpers.tpl

```diff
@@ -23,10 +23,13 @@
 {{- end -}}
 
 {{- define "rkConnect.url" }}
-{{- $rlsname := .Release.Name -}}
-{{- $namespace := .Release.Namespace -}}
 {{- $port := index .Values "riak" "ClientPortProtobuf" | toString -}}
-    rk-{{$rlsname}}.{{$namespace}}.svc:{{$port}}
+{{- range $num, $e := until (.Values.riak.replicas|int) -}}
+    {{- printf "rk-%s-%d.%s.svc:%d" $.Release.Name $num $.Release.Namespace ($.Values.riak.ClientPortProtobuf|int) -}}
```

Comment:
This is missing the service name, e.g.
rk-\<release\>-0.rk-\<release\>.\<namespace\>.svc